### DISCUSSION TITLE
posts/%EC%B2%AB-%ED%8F%AC%EC%8A%A4/

### DISCUSSION THREAD

**Discussion by giscus** [E-Yujin/E-Yujin.github.io#3] [General]
# posts/%EC%B2%AB-%ED%8F%AC%EC%8A%A4/

A minimal, responsive and feature-rich Jekyll theme for technical writing.

http://localhost:4000/posts/%EC%B2%AB-%ED%8F%AC%EC%8A%A4/

**Comment by E-Yujin** [comment]
오

**Reply by E-Yujin** [reply]
와아 신기해요~
